### ISSUE TITLE
aws_lambda_event_source_mapping requires update for SQS sources

### ISSUE THREAD

**Issue by watchwithmike** [VladRassokhin/intellij-hcl#158]
### Prerequisites
 - [X] Ensure you have latest version of plugin installed
 - [X] Search for possible issue duplicates

### Installation details
 - [X] IDE version (`Help->About->Copy to Clipboard`)
```
IntelliJ IDEA 2018.2.2 (Ultimate Edition)
Build #IU-182.4129.33, built on August 21, 2018
Licensed to Watchwith / Michael Dalrymple
Subscription is active until June 24, 2019
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-33-generic
```
 - [X] intellij-hcl plugin version (`Settings->Plugins`)
`Version: 0.6.11`
 - [X] Terraform version (`terraform -v`)
`Terraform v0.11.8`
### Terraform Configuration Files
```hcl
resource "aws_lambda_event_source_mapping" "sqs_queue_trigger" {
  event_source_arn  = "${aws_sqs_queue.extension_requests_queue.arn}"
  function_name     = "${aws_lambda_function.queue_to_queue.function_name}"
}
```

### Exception
```text
Missing required properties: starting_position less... (Ctrl+F1) 
Inspection info: Reports blocks with missing required properties
```

### Expected Behavior
starting_position is not required for SQS triggers

### Actual Behavior
Highlights the block as if it's an error

### Steps to Reproduce
1. Create an `aws_sqs_queue` resource
1. Create an `aws_lambda_function` resource.
1. Create an `aws_lambda_event_source_mapping` with the queue as the source and the lambda as the function.
Observe validation error.

[Terraform Docs](https://www.terraform.io/docs/providers/aws/r/lambda_event_source_mapping.html)

> starting_position - (Optional) The position in the stream where AWS Lambda should start reading. Must be one of either TRIM_HORIZON or LATEST if getting events from Kinesis or DynamoDB. Must not be provided if getting events from SQS.

**Comment by VladRassokhin** [comment]
Fixed in 0.6.12

**Comment by watchwithmike** [comment]
Confirmed.  Nice work as always @VladRassokhin, thanks!
